deployment/policies/ecs-instance-role-policy.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
A canned version of this policy should already exist within AWS as `arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role`. Using that reference is preferable because if they ever extend the functionality, we'll inherit it automatically.